pkg/shortnames/shortnames_test.go

```diff
@@ -479,13 +505,25 @@ func TestResolveLocally(t *testing.T) {
 	assert.Equal(t, "quay.io/repo/image:latest", aliases[2].String())      // registry 0
 	assert.Equal(t, "registry.com/repo/image:latest", aliases[3].String()) // registry 0
 
+	aliases, err = ResolveLocally(sysResolveToDockerHub, "repo/image") // alias match but enforced
+	require.NoError(t, err)
+	require.Len(t, aliases, 2)                                          // localhost + docker.io enforced
+	assert.Equal(t, "localhost/repo/image:latest", aliases[0].String()) // localhost
+	assert.Equal(t, "docker.io/repo/image:latest", aliases[1].String()) // docker.io enforced
+
```

Comment:
It seems to me that turning this long repetitive function into a table-driven test, with a `[]struct { input string; expected []string }`, is warranted enough to be a review blocker at this point, though I don’t feel _extremely_ strongly about it.